tests/scripts/create-gke-cluster.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+source scripts/vars.env
+
+ip_random_digit=$((1 + $RANDOM % 250))
+
+gcloud container clusters create ${GKE_CLUSTER_NAME} \
+    --project ${GKE_PROJECT} \
+    --zone ${GKE_CLUSTER_ZONE} \
+    --enable-master-authorized-networks \
+    --enable-ip-alias \
+    --service-account ${GKE_NODES_SERVICE_ACCOUNT} \
+    --enable-private-nodes \
+    --master-ipv4-cidr 172.16.${ip_random_digit}.32/28 \
+    --metadata=block-project-ssh-keys=TRUE
```

Comment:
Do we need `--master-authorized-networks`?

Also, do we need to set any `--tags` to link the VM to the cluster?

Reply:
`--master-authorized-networks` restricts who can access the control node (in this case, the script will allow cluster access from only the VM we are running the tests are). It's security best practices. 

> Also, do we need to set any --tags to link the VM to the cluster?

No, the network tags we specify in the vars.env are for network access from the local machine (or runner in this case) to the VM. The VM doesn't require this, as instead we are using `--master-authorized-networks` to restrict access based on IP.